aspnetcore/mvc/controllers/routing.md

```diff
@@ -563,8 +560,6 @@ Attribute routes support the same inline syntax as conventional routes to specif
 
 In the preceding code, `[HttpPost("product/{id:int}")]` applies a route constraint. The `ProductsController.ShowProduct` action is matched only by URL paths like `/product/3`. The route template portion `{id:int}` constrains that segment to only integers.
 
-[!code-csharp[](routing/samples/3.x/main/Controllers/HomeController.cs?name=snippet24)]
```

Comment:
I think this was an error, as the code being shown appears to have no relevance to the discussion around it. This also solves the original issue, where `snippet24` exists twice.